### PR TITLE
feat: support sm90 cutlass group gemm

### PR DESCRIPTION
**Pull request by xslingcn**

### CODE REVIEW

File: flashinfer-aot/csrc_aot/flashinfer_ops_sm90.cu [hunk 1]
```diff
@@ -13,143 +13,14 @@
...
-  m.def("gelu_and_mul", &gelu_and_mul, "Fused GeLU and Mul");
-  m.def("apply_rope_inplace", &apply_rope_inplace, "Apply RoPE in-place");
-  m.def("apply_llama31_rope_inplace", &apply_llama31_rope_inplace,
-        "Apply Llama 3.1 style RoPE in-place");
-  m.def("apply_rope", &apply_rope, "Apply RoPE");
-  m.def("apply_llama31_rope", &apply_llama31_rope, "Apply Llama 3.1 style RoPE");
-  m.def("packbits", &packbits, "GPU packbits operator");
-  m.def("segment_packbits", &segment_packbits, "GPU segment packbits operator");
   m.def("cutlass_segment_gemm_sm90", &CutlassSegmentGEMMSM90, "Cutlass Segment GEMM operator for SM90");
-  m.def("bmm_fp8", &bmm_fp8, "BMM FP8");
```
Comment (zhyncs) [inline flashinfer-aot/csrc_aot/flashinfer_ops_sm90.cu]: QQ Why do you remove this? @yzh119

Comment (yzh119) [reply flashinfer-aot/csrc_aot/flashinfer_ops_sm90.cu]: Compiling `bmm_fp8` do not require sm90 (the binary was already shipped in cublas). It was compiled in `flashinfer_ops.cu`.

Comment (zhyncs) [reply flashinfer-aot/csrc_aot/flashinfer_ops_sm90.cu]: Oh make sense. Thanks!

**Review by yzh119 — APPROVED** [review]

**Comment by zhyncs** [comment]
Amazing work! cc @xslingcn @yzh119

**Pull request merged**